### ISSUE TITLE
Dropdown menus react with delayed/sticked 

### ISSUE THREAD

**Issue by Alewx** [girka2k/UbioWeldContinued#33]
the current dropdown menus react delayed when the user switches to another dropdown menu, a behavior that feels broken.

**Comment by girka2k** [comment]
Confirmed, I also noticed this

**Comment by Alewx** [comment]
I'm working on this.

**Comment by Alewx** [comment]
current state is: visible dropdowns without logic and position problems in
https://github.com/Alewx/unofficailUbioWeld/commit/bf5242fd08e30d555b9b60a5e5722d1bbf8059c2

**Comment by girka2k** [comment]
strange effect with https://github.com/Alewx/unofficailUbioWeld/commit/bf5242fd08e30d555b9b60a5e5722d1bbf8059c2
Immideatly after entering VAB, I see button with label "10":
![2014-11-09 22-25-04 kerbal space program](https://cloud.githubusercontent.com/assets/6450733/4968654/ca41d674-6846-11e4-9b6e-0e35282d7f2c.png)
After click on "10" button there are the little buttons "Prev"/"Next" which change part's catalog pages
![2014-11-09 22-25-18 kerbal space program](https://cloud.githubusercontent.com/assets/6450733/4968657/fcb9ab36-6846-11e4-8c34-f3a60fb4e2b7.png)
The behavior of the dropdown lists has not changed :)

**Comment by Alewx** [comment]
Yep that is the problem I currently have, and the reason why I didn't merge it into the development branch.
Visible directly at entering the VAB for debuging purpose.
And where the hell these next and prev buttuons are from, is the real question. still under full development.
